### ISSUE TITLE
[TODO] Create a Features/Gallery Page

### ISSUE THREAD

**Issue by odysseywestra** [mypaint/mypaint.github.io#8]
Besides the main landing page, we need to create a dedicated features page that features more of the key features that MyPaint has to offer. A key example would be [krita's features page](https://krita.org/features/highlights/) which not only highlight key features, but also has a gallery section to features artwork that was created by Krita. The whole point of this page is not only win them over to use mypant, but also a place where new artists can draw inspiration from by following artist that are featured in the gallery.

**Comment by odysseywestra** [comment]
Okay this is for those who came from our [State of MyPaint.Org](http://mypaint.org/blog/2015/12/05/state-of-website/) post on our website. 

If you would like your artwork in MyPaint.org gallery, we will need the following information. 
* Your Name
* Art Piece Title
* Link to Art Piece

Also like state in the blog post, they will have to be in a creative common license to be compatible to our site. We just need the ability to adapt the work(For thumbnailing) and be able to use it for non-commercial use. Please post your best work and no WIPs please. Thank You.

**Comment by achadwick** [comment]
For clarification: we probably want a selection of screenshots of the program in use as well, but this issue is about collecting great examples of the program in the world. I'll mention screenshots in #15, since screenshots ≅ explanations of features.

**Comment by achadwick** [comment]
@odysseywestra Do we always need to host the image here ourselves? If not, maybe we could suggest relaxing the license constraint to "you only have to allow us to distribute a large, good-looking thumbnail under (preferred license here)", and just linking to the artist's own gallery page for the work. Let me know if you think that's unworkable.

Right now, I tentatively think we have to be able to use images hosted on the site for _commercial_ purposes if we choose. And so do people downloading data from the site. We probably won't do that, but it's a key requirement of the [DFSG](https://en.wikipedia.org/wiki/Debian_Free_Software_Guidelines) and other codes for free licensing that _commercial_ reuse of program documentation material is permitted. Documentation could include things like a website, if snapshotting the site were to form part of the official docs.

Artists may only want us to use a thumbnail on our website, so they can distribute the real deal under their own license. And that's probably the best compromise position if there are any issues.

-------------------

For everyone's reference: the MyPaint licensing policy covers promotional material: <https://github.com/mypaint/mypaint/wiki/Licensing-policy#supplemental-artwork-or-promotional-material>. But it only covers stuff we ourselves distribute as part of the program download really.

(Of course, there is cross-pollenization allowed: promo art on the site could become in-program textures, splash screens etc. That's the point of having free licenses.)

**Comment by odysseywestra** [comment]
Yeah, My initial idea of of creating the gallery was to use the thumbnail to create a masonry layout and just have those thumbnails link to the urls the artists provide. So basially we are on the same page.

I'll edit the fist post to include to include the licensing policy wiki page. Thank you for pointing that out.

**Comment by odysseywestra** [comment]
@achadwick Opened a Discussion Thread about needing screenshots on our Community Fourms. You can post them there as well. 

http://community.mypaint.org/t/screenshots-of-1-2-0-during-development/44?u=odysseywestra
